### ISSUE TITLE
Switch 13.1.0 Version not working

### ISSUE THREAD

**Issue by MeirBelach** [olliz0r/sys-botbase#49]
I have CFW 13.1.0 on AMS 1.2.4 and when I install the current package (Following the instructions); I can't get the current sys-botbase to activate. I was was wondering is there a 13.1.0 version coming or am I doing something wrong?

Also, when I check the sysmodules it is not showing up.

Any guidance would be appreciated.

Thank You

**Comment by berichan** [comment]
No changes in 13.1.0 would affect functionality of sys-botbase. It's working like normal

**Comment by MeirBelach** [comment]
I figured out how to get the module activated on the switch. But, trying to connect to it I get the error along the lines "Connection Refused". Is there something else I have to do?

**Comment by EXO5KELETON** [comment]
Ok so on the new version something causes atmosphere to crash.

**Comment by EXO5KELETON** [comment]
So the problem seems to be with either Tesla-menu or nx-ovlloader...Soon as I create a new contents folder it works...does this mean either one of these has to be updated for some reason?

**Comment by Bladeage** [comment]
Does someone has an answer to that?
I'm still crashing with nx-ovlloader or Tesla activated and cannot access hbmenu while sys-botbase as an active sysmodule (result = crash).

**Comment by EXO5KELETON** [comment]
Yes I solved this… I uninstalled all my sys modules installed sys-botbase and the rebooted and then reinstalled all my sys modules
